docs/ADD_NEW_EXCHANGE.md

```diff
@@ -0,0 +1,1047 @@
+# GoCryptoTrader package Exchanges
+
+<img src="https://github.com/thrasher-corp/gocryptotrader/blob/master/web/src/assets/page-logo.png?raw=true" width="350px" height="350px" hspace="70">
+
+[![Build Status](https://travis-ci.org/thrasher-corp/gocryptotrader.svg?branch=master)](https://travis-ci.org/thrasher-corp/gocryptotrader)
+[![Software License](https://img.shields.io/badge/License-MIT-orange.svg?style=flat-square)](https://github.com/thrasher-corp/gocryptotrader/blob/master/LICENSE)
+[![GoDoc](https://godoc.org/github.com/thrasher-corp/gocryptotrader?status.svg)](https://godoc.org/github.com/thrasher-corp/gocryptotrader/exchanges)
+[![Coverage Status](http://codecov.io/github/thrasher-corp/gocryptotrader/coverage.svg?branch=master)](http://codecov.io/github/thrasher-corp/gocryptotrader?branch=master)
+[![Go Report Card](https://goreportcard.com/badge/github.com/thrasher-corp/gocryptotrader)](https://goreportcard.com/report/github.com/thrasher-corp/gocryptotrader)
+
+This exchanges package is part of the GoCryptoTrader codebase.
+
+## This is still in active development
+
+You can track ideas, planned features and what's in progress on this Trello board: [https://trello.com/b/ZAhMhpOy/gocryptotrader](https://trello.com/b/ZAhMhpOy/gocryptotrader).
+
+Join our slack to discuss all things related to GoCryptoTrader! [GoCryptoTrader Slack](https://join.slack.com/t/gocryptotrader/shared_invite/enQtNTQ5NDAxMjA2Mjc5LTc5ZDE1ZTNiOGM3ZGMyMmY1NTAxYWZhODE0MWM5N2JlZDk1NDU0YTViYzk4NTk3OTRiMDQzNGQ1YTc4YmRlMTk)
+
+## Current Features for exchanges
+
++ This package is used to connect and query data from supported exchanges.
+
++ Please checkout individual exchange README for more information on
+implementation
+
+## How to add a new exchange
+
+This document is from a perspective of adding a new exchange called FTX to the codebase:
+
+### Run exchange_template.go which automatically creates files & inbuilt functions
+
+#### Linux/OSX
+GoCryptoTrader is built using [Go Modules](https://github.com/golang/go/wiki/Modules) and requires Go 1.11 or above
+Using Go Modules you now clone this repository **outside** your GOPATH
+
+```bash
+git clone https://github.com/thrasher-corp/gocryptotrader.git
+cd exchange_template
+go run exchange_template.go -name FTX -ws -rest
+```
+
+#### Windows
+
+```bash
+git clone https://github.com/thrasher-corp/gocryptotrader.git
+cd gocryptotrader\cmd\exchange_template
+go run exchange_template.go -name FTX -ws -rest
+```
+
+### Add exchange struct to config_example.json, configtest.json (in testdata) & to the main config:
```

Comment:
Would be nice to link to these files `(config_example.json)[../config_example.json]` and `(configtest.json)[../testdata/config_example.json]` plus can rid the "& to the main config"